frameworks/core_foundation/views/view/enabled.js

```diff
@@ -204,6 +204,7 @@ SC.View.reopen(
     // Update the display if in a visible state.
     switch (state) {
     case SC.CoreView.ATTACHED_SHOWN:
+    case SC.CoreView.ATTACHED_SHOWN_ANIMATING:
```

Comment:
Not about the line itself, but I notice this is a fall-through. I think it would be better to replace this switch by doing something like:
``` 
var isVisible = [ 
  SC.CoreView.ATTACHED_SHOWN, 
  SC.CoreView.ATTACHED_SHOWN_ANIMATING,
  SC.CoreView.ATTACHED_SHOWING,
  SC.CoreView.ATTACHED_BUILDING_IN].indexOf(state) > -1;

if (isVisible) {
  this._doUpdateEnabledStyle();
} 
else this._enabledStyleNeedsUpdate = true;
```